third_party/CMakeLists.txt

```diff
@@ -83,13 +83,19 @@ if (SHADERC_ENABLE_SPVC)
     if (IS_DIRECTORY ${SHADERC_SPIRV_CROSS_DIR})
       set(SPIRV_CROSS_EXCEPTIONS_TO_ASSERTIONS ${DISABLE_EXCEPTIONS} CACHE BOOL "tell SPIRV-Cross not to throw" FORCE)
 
-      # Add -fPIC to SPIRV-Cross build.
-      set(CXX_BACK ${CMAKE_CXX_FLAGS})
-      set(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS} "-fPIC")
-      # cmake inserts a semicolon, change it to a space.
-      string(REGEX REPLACE ";" " " CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
-      add_subdirectory(${SHADERC_SPIRV_CROSS_DIR} spirv-cross)
-      set(CMAKE_CXX_FLAGS ${CXX_BACK})
+      # Add -fPIC to SPIRV-Cross build, if supported
+      unset(COMPILER_SUPPORTS_PIC CACHE)
+      check_cxx_compiler_flag(-fPIC COMPILER_SUPPORTS_PIC_WARNING)
```

Comment:
Drop the _WARNING suffix here.  It doesn't match the unset on the previous line.

Reply:
Done